04-account-kit-manage-permissions/index.ts

```diff
@@ -0,0 +1,48 @@
+// SDK exposes all you need
+import { Chains, PublicKey, SigningRequest, Permission } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account
+    const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Manage account Permissions
+
+    // Get a permission
+    const permission: Permission = account.permissions.get("active")
+
+    // Get its public keys
+    const publicKeys: PublicKey[] = permission.keys
+
+    // Get its accounts
+    const accounts: Account[] = permission.accounts
+
+    // Add a new permission
+    const addPermissionEsr: SigningRequest = account.permissions.add('active', PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY'))
+
+    // Update a new permission
+    const updatePermissionEsr: SigningRequest = account.permissions.get('active').update(PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY'))
```

Comment:
Updating a permission will probably require some sort of specificity on top of this. 

I think the `update` call probably will require passing in an entire permission, and it'll replace the permission with those values? 

In the above code, I don't know if it's adding a public key to the keys of the permission, or replacing, or what exactly it's doing. The `update` method should probably also include accounts/waits/etc too.

Reply:
Yeah, good callout!